src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let basePrice = days * 40;
```

Comment:
Prefer `const` over `let` for `basePrice` since its value is not being reassigned. This helps to prevent unintentional changes to the variable.